src/components/application_manager/include/application_manager/hmi_capabilities.h

```diff
@@ -416,6 +416,13 @@ class HMICapabilities {
 
   void Init(resumption::LastState* last_state);
 
+  /**
+   * @brief Removes application from container after
+   * removing application from core
+   * @param app_id id application for removing
+   */
+  void OnUnregisterApplication(uint32_t app_id);
```

Comment:
@dtrunov 
I don't like that interface - it is inconsistent with purpose of HMI Capabilities class